src/helm/benchmark/static/schema.yaml

```diff
@@ -476,35 +476,43 @@ metrics:
     display_name: Exact match
     short_display_name: EM
     description: Fraction of instances that the predicted output matches a correct reference exactly.
+    lower_is_better: False
```

Comment:
minor: noticed the existing annotations use "true" (i.e. little "t"), but this says "False" (i.e. big "F"). Is this intentional?